page.go

```diff
@@ -224,26 +224,23 @@ func (p *Page) UnmarshalJSON(b []byte) error {
 }
 
 func (p UpdatePageParams) Validate() error {
-	if p.DatabasePageProperties == nil && p.Title == nil {
-		return errors.New("either database page properties or title is required")
+	if p.DatabasePageProperties == nil {
```

Comment:
The `properties` param is optional, and thus shouldn't be required by itself. This `if` statement should become an equality comparison on the empty value of `UpdatePageParams`, e.g.:
```suggestion
	if p == (UpdatePageParams{}) {
```